keps/sig-node/3857-rro-mounts/README.md

```diff
@@ -693,9 +693,12 @@ in back-to-back releases.
 
 #### Beta
 - e2e tests pass with containerd, CRI-O, and cri-dockerd
+ - https://github.com/containerd/containerd/pull/9787
+ - https://github.com/cri-o/cri-o/pull/7962
+ - https://github.com/Mirantis/cri-dockerd/pull/370
 
 #### GA
-- (Will be revisited during beta)
+- Two beta releases at least
 
 ### Upgrade / Downgrade Strategy
```

Comment:
I'm missing the following sections to be filled in:
- current unit test coverage
- integration and e2e tests, with links to testgrid, where possible. 
- in `###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?` you're talking this will be revisited during beta, are you planning the upgrade/downgrade tests? 
- update `## Implementation History`

Reply:
Fulfilled.